client_integration_test.go

```diff
@@ -1489,6 +1489,49 @@ func TestClientReadFrom(t *testing.T) {
 	}
 }
 
+func TestClientReadFromSectionReader(t *testing.T) {
+	const (
+		packetSize = 1024
+		bufsize    = 4 * packetSize
+		skip       = 100
+		filesize   = bufsize - skip
+	)
+
+	sftp, cmd := testClient(t, READWRITE, NODELAY,
+		MaxPacketChecked(packetSize),
+		UseConcurrentWrites(true))
+	defer cmd.Wait()
+	defer sftp.Close()
+
+	d, err := ioutil.TempDir("", "sftptest-readfrom-sectionreader")
+	if err != nil {
+		t.Fatal(err)
```

Comment:
I recommend a bit of additional information as to why the fatal is being called: `t.Fatal("unexpected error:", err)`

Reply:
Done.